.env.production.sample

```diff
@@ -90,3 +90,8 @@ SMTP_FROM_ADDRESS=notifications@example.com
 # Cluster number setting for streaming API server.
 # If you comment out following line, cluster number will be `numOfCpuCores - 1`.
 STREAMING_CLUSTER_NUM=1
+
+# Piwik ( Optional )
+# PIWIK_ENABLED=true
```

Comment:
This can probably be condensed down to two variables, no? if there is a piwik domain, then enable piwik, otherwise don't.